doc/sphinx/source/users-guide/index.rst

```diff
@@ -0,0 +1,40 @@
+.. _users-guide-index:
+
+User's Guide
+============
```

Comment:
It might nice to have a disclaimer on the users guide landing page, explicitly stating that each section of the guide is currently a work in progress.